messages/context.json

```
@@ -74,5 +74,12 @@
   "docs-ui/join-discussion": "Join the discussion!",
   "docs-ui/did-not-find-your-answer": "Didn't find your answer?",
   "docs-ui/lorem": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam vel ligula sagittis, dignissim nisi non, interdum elit. Vivamus eget dolor egestas, hendrerit urna a, finibus enim. Phasellus mollis mollis dui ut mattis. In ut purus sem. Quisque volutpat ornare consequat.",
-  "docs-ui/lorem-short": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod."
+  "docs-ui/lorem-short": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.",
+  "DocProp.table.title": "Name",
```

```suggestion
  "docs-ui/docprop.table.title": "Name",
```

Both the casing and scope should be fixed (for all messages)